### PR TITLE
Tests: PAM can't identify the user when running from external host

### PR DESCRIPTION
**Pull request by aborah-sudo**
https://issues.redhat.com/browse/RHEL-16727

### CODE REVIEW

File: multihost_test/bz_automation/test_chkpwd-onlyroot.py [hunk 1]
```diff
@@ -156,3 +157,45 @@ def test_19810(self, multihost, bkp_pam_config):
...
         client.run_command("faillock")
         assert "Password: su: Authentication failure" not in result1.readlines()[0]
+
+    def test_16727(self, multihost, bkp_pam_config, create_localusers):
+        """
+        :title: PAM can't identify the user when running from external host
+        :id: ae78e4d6-ba95-11ee-8362-845cf3eff344
+        :bugzilla: https://issues.redhat.com/browse/RHEL-16727
+        :steps:
+          1. Create "testuser" user and set password
```
Comment (ikerexxe) [inline multihost_test/bz_automation/test_chkpwd-onlyroot.py:167]: I don't see any code for this step

File: multihost_test/bz_automation/test_chkpwd-onlyroot.py [hunk 2]
```diff
@@ -156,3 +157,45 @@ def test_19810(self, multihost, bkp_pam_config):
...
+          2. Add user to users group
```
Comment (ikerexxe) [inline multihost_test/bz_automation/test_chkpwd-onlyroot.py:168]: s/users/testgroup. At least that's how you call it in the code.

File: multihost_test/bz_automation/test_chkpwd-onlyroot.py [hunk 3]
```diff
@@ -156,3 +157,45 @@ def test_19810(self, multihost, bkp_pam_config):
...
+          3. Configure "testuser" user in sudoers to be able to sudo without password
+          4. Configure /etc/pam.d/sudo
+          5. Configure /etc/pam.d/su
+          6. ssh into the machine as "testuser" and issue "sudo su"
```
Comment (ikerexxe) [inline multihost_test/bz_automation/test_chkpwd-onlyroot.py:172]: s/testuser/local_anuj

File: multihost_test/bz_automation/test_chkpwd-onlyroot.py [hunk 4]
```diff
@@ -156,3 +157,45 @@ def test_19810(self, multihost, bkp_pam_config):
...
+                           "pam_wheel.so trust group=testgroup debug' /etc/pam.d/su")
+        client.run_command("> /var/log/secure")
+        time.sleep(1)
+        ssh1 = SSHClient(multihost.client[0].ip, username="local_anuj", password="password123")
+        ssh1.execute_cmd('sudo su -c "ls /root/"')
+        time.sleep(1)
+        assert "Access granted to 'local_anuj' for 'root'" \
+               in client.run_command("cat /var/log/secure").stdout_text
+        assert "Access denied to 'root' for 'root'" \
+               not in client.run_command("cat /var/log/secure").stdout_text
```
Comment (ikerexxe) [inline multihost_test/bz_automation/test_chkpwd-onlyroot.py:201]: I don't think this is needed. In addition, it isn't explained in the comments.

**Review by ikerexxe — COMMENTED** [review]
Some inline comments.

File: multihost_test/bz_automation/test_chkpwd-onlyroot.py (hunk 1 above)
Comment (aborah-sudo) [reply multihost_test/bz_automation/test_chkpwd-onlyroot.py:167]: This is with fixture create_localusers

Comment (ikerexxe) [reply multihost_test/bz_automation/test_chkpwd-onlyroot.py]: You are right. I always forget about the fixtures :facepalm:

**Review by ikerexxe — APPROVED** [review]
LGTM!

**Pull request merged**